src/components/Button/Button.styled.ts

```diff
@@ -54,13 +56,12 @@ export const SButton = styled.button<ButtonProps>`
     `}
 `
 
-export const SContent = styled.span`
+export const SContent = styled.span<{ size: ButtonSize }>`
   display: flex;
-  gap: 10px;
+  gap: ${({ size }) => (size === "compact" || size === "micro" ? "4" : "10")}px;
```

Comment:
Can you add all Button sizes to Storybook?

Also, maybe we should rework Button component and rename the sizes values to something more logical.
Currently we have compact / micro / small,  which are justs different words that mean small button.

We should rework it to small / medium / large, like we have in other components.
Can also add extra-small or extra-large if needed.

You dont have to do it in this PR, but I think it would make sense to refactor it in some future update.

Reply:
We can do it in a different PR